ldi-core/geojson-to-wkt/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/GeoJsonToWktTransformer.java

```diff
@@ -0,0 +1,67 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiTransformer;
+import org.apache.jena.geosparql.implementation.datatype.WKTDatatype;
+import org.apache.jena.rdf.model.Literal;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.WktConverter.GEOJSON_GEOMETRY;
+
+public class GeoJsonToWktTransformer implements LdiTransformer {
+
+	private final WktConverter wktConverter = new WktConverter();
+
+	/**
+	 * Replaces all geojson:geometry statements with locn:geometry statements
+	 * containing geosparql#wktLiteral
+	 */
+	@Override
+	public Model apply(Model model) {
+		final List<Statement> geometryStatements = model.listStatements(null, GEOJSON_GEOMETRY, (RDFNode) null)
+				.toList();
+		geometryStatements.forEach(oldGeometryStatement -> {
```

Comment:
This foreach seems to contract with line 34 of WKTConverter: `if (geometryStatements.size() > 1)`